src/main/resources/i18n/Scalars_de.properties

```diff
@@ -10,14 +10,11 @@
 # REMEMBER - a single quote ' in MessageFormat means things that are never replaced within them
 # so use 2 '' characters to make it one ' on output.  This will take for the form ''{0}''
 #
-# Prior to Java 9, properties files are encoded in ISO-8859-1.
-# We have to use \u00fc instead of the German ue character, \u00e4 for ae, \u00f6 for oe, \u00df for ss
-#
 Scalar.unexpectedAstType=Erwartet wurde ein AST type von ''{0}'', aber es war ein ''{1}''
 #
-Enum.badInput=Ung\u00fcltige Eingabe f\u00fcr enum ''{0}''. Unbekannter Wert ''{1}''
-Enum.badName=Ung\u00fcltige Eingabe f\u00fcr enum ''{0}''. Kein Wert f\u00fcr den Namen ''{1}'' gefunden
-Enum.unallowableValue=Literal nicht in den zul\u00e4ssigen Werten f\u00fcr enum ''{0}'' - ''{1}''
+Enum.badInput=Ung�ltige Eingabe f�r enum ''{0}''. Unbekannter Wert ''{1}''
+Enum.badName=Ung�ltige Eingabe f�r enum ''{0}''. Kein Wert f�r den Namen ''{1}'' gefunden
+Enum.unallowableValue=Literal nicht in den zul�ssigen Werten f�r enum ''{0}'' - ''{1}''
```

Comment:
See the umlauts rendered correctly at https://github.com/graphql-java/graphql-java/blob/0428ebb85c38a655a4e4b1d3bfa897bbd48b94b0/src/main/resources/i18n/Scalars_de.properties